recipe/meta.yaml

```
@@ -50,13 +50,17 @@ source:
       # https://github.com/tensorflow/tensorflow/pull/86413
       - patches/0029-Avoid-linking-with-internal-nvrtc.patch
       - patches/0030-remove-dependencies-to-libcuda.patch
+      - patches/0031-bump-h5py-req.patch
+      - patches/0032-gpu_prim-error.patch
```

Can you add a comment where this patch comes from? e.g.
```suggestion
      # backport https://github.com/tensorflow/tensorflow/pull/xyz
      - patches/0032-gpu_prim-error.patch
```

---

Thanks! Yep.